src/screens/globalNotifications.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from 'react'
+import { View, FlatList, StyleSheet, Text, RefreshControl } from 'react-native'
+import { ActivityIndicator, Caption, Divider } from 'react-native-paper'
+import { useQuery } from 'urql'
+import { NOTIFICATIONS } from '../api/gql'
+import { Notification } from '../api/types'
+import colors from '../utils/colors'
+import { globalStyles } from '../utils/styles'
+
+
+export const GlobalNotifications = () => {
+    const [refreshing, setRefreshing] = useState(false);
+
+    type NotificationsResult = { notifications: Array<Notification> }
+
+    const [notifcationResponse, getNotifications] = useQuery<NotificationsResult>({
+        query: NOTIFICATIONS,
+    })
+
+    const notifications = notifcationResponse.data?.notifications
+
+    useEffect(() => {
+        if (notifcationResponse.data?.notifications) {
+            setRefreshing(false)
+        } else if (notifcationResponse.error) {
+            setRefreshing(false)
+        }
+    }, [notifcationResponse])
+
+    const onRefresh = () => {
+        setRefreshing(true)
+        getNotifications({ requestPolicy: 'network-only' })
+    }
+
+    const renderNotificatonItem = ({ item }: { item: Notification }) => {
+
+        return (
+            <>
+                <View style={styles.notification}>
+                    <Text style={styles.notificationTitle}>{item.name}</Text>
+                    <Text style={styles.notificationSubTitle}>{item.description}</Text>
+                    {item.time && <Caption>{item.time}</Caption>}
```

Comment:
Can you format this like in Figma? i.e using relative dates until absolute w/ formatting

Reply:
Updated